<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-A942O01" failures="1" tests="5" name="Listeners Test" time="4.742" errors="0" timestamp="2025-04-10T14:32:11 IST">
  <testcase classname="tests.LoginTest" name="failingOnPurpose" time="0.068">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.LoginTest.failingOnPurpose(LoginTest.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- failingOnPurpose -->
  <testcase classname="tests.LoginTest" name="testLogin" time="1.066"/>
  <testcase classname="tests.LoginTest" name="testLogin" time="0.52"/>
  <testcase classname="tests.LoginTest" name="testLogin" time="0.529"/>
  <testcase classname="tests.LoginTest" name="testLogin" time="0.628"/>
</testsuite> <!-- Listeners Test -->
